### ISSUE TITLE
ROUTER_NAVIGATED is missing in @ngrx/router-store v6.1.2

### ISSUE THREAD

**Issue by vagonpidarasov** [ngrx/platform#1407]
## Minimal reproduction of the bug/regression with instructions:

[@ngrx/router-store@6.1.2](https://www.npmjs.com/package/@ngrx/router-store/v/6.1.2) installed via npm doesn't have `ROUTER_NAVIGATED` action.

## Expected behavior:
It is possible to import ROUTER_NAVIGATED from @ngrx/router-store:
`import {ROUTER_NAVIGATED} from '@ngrx/router-store';`

## Versions of NgRx, Angular, Node, affected browser(s) and operating system(s):
```
    "@angular/animations": "6.1.0",
    "@angular/cdk": "6.4.2",
    "@angular/cli": "6.1.2",
    "@angular/common": "6.1.0",
    "@angular/compiler": "6.1.0",
    "@angular/compiler-cli": "6.1.0",
    "@angular/core": "6.1.0",
    "@angular/forms": "6.1.0",
    "@angular/http": "6.1.0",
    "@angular/material": "6.3.0",
    "@angular/platform-browser": "6.1.0",
    "@angular/platform-browser-dynamic": "6.1.0",
    "@angular/router": "6.1.0",
    "@ngrx/effects": "6.1.0",
    "@ngrx/router-store": "6.1.2",
    "@ngrx/store": "6.1.0",
```

## I would be willing to submit a PR to fix this issue

[ ] Yes (Assistance is provided if you need help submitting a pull request)
[x] No

**Comment by timdeschryver** [comment]
The `ROUTER_NAVIGATED` action is [introduced](https://github.com/ngrx/platform/blob/master/CHANGELOG.md#700-beta0-2018-11-03) in `7.0.0-beta.0`, so this is not available in `6.1.2`.

The docs reflect the documentation for `7.0.0-beta.0`.

**Comment by sbkpilot1** [comment]
I noticed that the payload in `ROUTER_NAVIGATED ` does not have the router state but only the event.  I need to query the Router State to obtain data associated with the route.  Any info on why this was omitted?
